Source/WebCore/css/mathml.css

```diff
@@ -139,7 +139,7 @@ math {
 }
 
 math[display="block" i] {
-    display: block;
+    display: block math;
     page-break-inside: avoid;
     math-style: normal;
 }
```

Comment:
Can probably be in a separate PR, but https://www.w3.org/TR/mathml-core/#user-agent-stylesheet also has `math[display="inline" i] { display: inline math; }` (not sure whether this is tested/testable).